### ISSUE TITLE
[Bug Report] Checkbox label does not have a corresponding `for` attribute

### ISSUE THREAD

**Issue by deadbeef404** [vuetifyjs/vuetify#6178]
### Versions and Environment
**Vuetify:** 1.4.2
**Vue:** 2.5.18
**Browsers:** Chrome 70.0.3538.110
**OS:** Mac OS 10.11.6

### Steps to reproduce
Inspect the `label` element for the checkbox.

### Expected Behavior
The `label` element should have a `for` attribute pointing to the id of the checkbox `input` element. 

### Actual Behavior
There is no `for` attribute.

### Reproduction Link
<a href="https://codepen.io/deadbeef404/pen/ZVZNOJ" target="_blank">https://codepen.io/deadbeef404/pen/ZVZNOJ</a>

### Other comments
This is necessary for accessibility. I have created a fix, and will submit the PR shortly.

**Comment by Denizar** [comment]
The v-radio eleent also does not have a for attribute, @deadbeef404  could you also check this component and replicate your fix if applicable?
 
Thanks

**Comment by Denizar** [comment]
The v-radio passes its id to the labels for attribute, but the id is not defined in the props as far as I can tell.
 
In any case currently in ios mobile browsers & cordova builds clicking on the labels do not work - hope adding the for attribute fixes this issue.
 
Cheers

**Comment by johnleider** [comment]
This also applies to any input that extends input and overwrites **genLabel**.

**Comment by deadbeef404** [comment]
> The v-radio element also does not have a for attribute, @deadbeef404 could you also check this component and replicate your fix if applicable?
> 
> Thanks

 @Denizar Restored `for`/`id` attribute pairs for radio buttons in PR#6235 as shown above.

**Comment by johnleider** [comment]
Fix reverted in 666e0fc49b233af98f5d11d3bfee995b26939893
